src/StorageManager.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * StorageManage.js
+ * ============
+ * Handles the persistence of data for the service relying on an in memory structure for now, but
+ * abstracted so we can move to a database in the future.
+ */
+class StorageManager {
```

Comment:
It seems like this class is named generically, but all of its functionality is specific to the departure delay data. Should this be a specific class, then?

If it is supposed to be generic, should it make use of the in-memory database, as seen in https://github.com/FerryTempo/FTServer/blob/main/src/App.js?

Reply:
I was leaving it generic for now because I figured there might be other data to store, and we might want to move the DB related code out of the main app into this class. What do you think about that idea?

Reply:
I think this PR is good to go for now. If we need to scale local storage later, we can explore the DB class.